apps/ledger-live-mobile/src/navigation/DeeplinksProvider.tsx

```diff
@@ -487,6 +487,17 @@ export const DeeplinksProvider = ({
             appName,
           } = query;
 
+          if (!ajsPropSource) {
+            /** Internal deep links cause the app to "background" on Android.
+             * If "password lock" is enabled then this opens a re-authentication modal every time the user navigates by deep link.
+             * If there is no ajsPropSource then assume deep link is from an internal app, so temporarily prevent password lock:
+             */
+            dispatch(blockPasswordLock(true)); // TODO: Remove this and the timeout after AuthPass refactor
```

Comment:
This is the main change. The rest is just renaming to make it clearer what the isDeepLinking functionality was actually doing, and why.